### PR TITLE
Vue translation for get-started.md

### PR DESCRIPTION
**Pull request by Ericnunez**
First translation ready for review

### CODE REVIEW

File: content/vue/es/get-started.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: "Empezando"
+tocTitle: "Empezando"
+description: "Configurar Vue Storybook en tu entorno de desarrollo"
+---
+
+# Empezando
+
+Storybook se ejecuta junto con tu aplicación en modo desarrollo. Te ayuda a crear componentes de UI aislados de la lógica y el contexto de su aplicación. Esta edición de Aprende Storybook es para Vue; otras ediciones existen para  [React](/react/es/get-started) y [Angular](/angular/es/get-started).
```
Comment (icarlossz) [inline content/vue/es/get-started.md]: I suggest `existen otras ediciones para [React](/react/es/get-started) y [Angular](/angular/es/get-started).`

File: content/vue/es/get-started.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+![Storybook and your app](/storybook-relationship.jpg)
+
+## Configurando Vue Storybook
+
+Tendremos que seguir algunos pasos para configurar el proceso de compilación en su entorno. Para empezar, vamos a querer usar [Vue CLI](https://cli.vuejs.org) para configurar nuestro sistema de compilación. También añadiremos[Storybook](https://storybook.js.org/) y [Jest](https://facebook.github.io/jest/) para probar nuestra aplicación. Vamos a ejecutar los siguientos comandos:
```
Comment (icarlossz) [inline content/vue/es/get-started.md]: * `en tu entorno`.

File: content/vue/es/get-started.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+cd taskbox
+
+# Añade Storybook:
+npx -p @storybook/cli sb init
+```
+
+Podemos comprobar rápidamente que los distintos entornos de nuestra aplicación funcionan correctamente:
+
+```bash
+# Ejecuta el corredor de prueba (Jest) en un terminal.
```
Comment (icarlossz) [inline content/vue/es/get-started.md]: `Ejecuta el compilador de prueba`

File: content/vue/es/get-started.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+yarn test:unit
+
+# Inicia el explorador de componentes en el puerto 6006:
+yarn run storybook
+
+# Ejecuta el frontend de la aplicación en el puerto 8080:
+yarn serve
+```
+<div class="aside">
```
Comment (icarlossz) [inline content/vue/es/get-started.md]: Why there html here? 😅

File: content/vue/es/get-started.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  NOTA: Si <code>yarn test:unit</code> produce un error, es posible que no tenga <a href="https://yarnpkg.com/lang/en/docs/install/">Yarn instalado</a> o es posible que debe instalar <code>watchman</code> como se indica en <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">este problema</a>.
+</div>
+
+Nuestras tres modalidades del frontend de la aplicación: test automatizado (Jest), desarrollo de componentes (Storybook) y la propia aplicación.
+
+![3 modalidades](/app-three-modalities.png)
+
+Dependiendo en qué parte de la aplicación en la que estés trabajando, es posible que quieras ejecutar uno o más de estos simultáneamente. Dado que nuestro enfoque actual es crear un único componente de UI, nos mantendremos en la ejecución de Storybook.
```
Comment (icarlossz) [inline content/vue/es/get-started.md]: I suggest `Dependiendo en qué parte de la aplicación estés trabajando`

File: content/vue/es/get-started.md (hunk 4 above)
Comment (Ericnunez) [reply content/vue/es/get-started.md]: The original translation has this html. I believe it is meant to provide special formatting when displayed on [https://www.learnstorybook.com/vue/en/get-started](url). I believe that it needs to be here.

**Comment by Ericnunez** [comment]
@icarlossz, I made the requested changes, let me know if I need to do anything else?

**Comment by domyen** [comment]
@Ericnunez are you planning on working on the rest of the chapters? Not sure if it makes sense to merge this one chapter at a time. What do you think @icarlossz?

**Comment by domyen** [comment]
This PR seems inactive so am closing to make maintenance easier.

**Pull request closed without merging**